client/site-profiler/components/get-report-form/index.tsx

```diff
@@ -98,89 +98,89 @@ export function GetReportForm( {
-								value={ email }
-								isError={ !! errors?.email }
-								onChange={ handleEmailChange }
-							/>
-							{ errors?.email && (
-								<FormInputValidation isError={ !! errors } text={ errors.email } />
+				<div className="get-report-form__body">
+					<div className="get-report-form__header">
+						<span className="description">
+							{ translate(
+								'Enter your details below to receive the full report with detailed insights and recommendations for your site.'
 							) }
-						</FormFieldset>
+						</span>
+						<span>
+							<Gridicon icon="cross" onClick={ onClose } />
+						</span>
 					</div>
-					<div className="get-report-form__form-footer">
-						<FormFieldset>
-							<CheckboxControl
-								name="termsAccepted"
-								className="terms-checkbox"
-								checked={ isTermsChecked }
-								onChange={ handleTermsChange }
-								label={ translate(
-									`By submitting your details, you agree to WordPress.com‘s Privacy Policy and Terms of Service. You also consent to receiving occasional updates and offers. You can unsubscribe from these communications at any time through the instructions.`
+					<form className="get-report-form__form" onSubmit={ handleSubmit }>
+						<div className="get-report-form__form-body">
+							<FormFieldset>
+								<FormLabel htmlFor="name">{ translate( 'Name' ) }</FormLabel>
+								<FormTextInput
+									name="name"
+									className="get-report-form__form-name"
+									label={ translate( 'Name' ) }
+									value={ name }
+									isError={ !! errors?.name }
+									onChange={ handleNameChange }
+								/>
+
+								{ errors?.name && (
+									<FormInputValidation isError={ !! errors } text={ errors.name } />
+								) }
+							</FormFieldset>
+							<FormFieldset>
+								<FormLabel htmlFor="email">{ translate( 'Email' ) }</FormLabel>
+								<FormTextInput
+									name="email"
+									className="get-report-form__form-email"
+									label={ translate( 'Email' ) }
+									value={ email }
+									isError={ !! errors?.email }
+									onChange={ handleEmailChange }
+								/>
+								{ errors?.email && (
+									<FormInputValidation isError={ !! errors } text={ errors.email } />
 								) }
+							</FormFieldset>
+						</div>
+						<div className="get-report-form__form-footer">
+							<FormFieldset>
+								<CheckboxControl
+									name="termsAccepted"
+									className="terms-checkbox"
+									checked={ isTermsChecked }
+									onChange={ handleTermsChange }
+									label={ translate(
+										`By submitting your details, you agree to WordPress.com‘s Privacy Policy and Terms of Service. You also consent to receiving occasional updates and offers. You can unsubscribe from these communications at any time through the instructions.`
+									) }
+								/>
+								{ errors?.termsAccepted && (
+									<FormInputValidation isError={ !! errors } text={ errors.termsAccepted } />
```

Comment:
Nit: Is this check `isError={ !! errors }` needed? Considering its checked on the previous line.
```suggestion
									<FormInputValidation isError text={ errors.termsAccepted } />
```

The same for the previous components.

Reply:
Not needed here, removing it

Reply:
3a474a8716